Assets/Thirdweb/Core/Scripts/Bridge.cs

```diff
@@ -91,7 +91,7 @@ public static async Task<string> Connect(WalletConnection walletConnection)
             string taskId = Guid.NewGuid().ToString();
             taskMap[taskId] = task;
 #if UNITY_WEBGL
-            ThirdwebConnect(taskId, walletConnection.provider.ToString(), walletConnection.chainId, walletConnection.password ?? Utils.GetDeviceIdentifier(), jsCallback);
+            ThirdwebConnect(taskId, walletConnection.provider.ToString().Substring(0,1).ToLower() + walletConnection.provider.ToString().Substring(1), walletConnection.chainId, walletConnection.password ?? Utils.GetDeviceIdentifier(), jsCallback);
```

Comment:
bit scary, but prob fine. Another thing we could do is just compare lowercased values in JS

Reply:
Yeah that'd be better